user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_Virtual_Hard_Disk.md

```diff
@@ -127,6 +127,39 @@ Follow the below steps to configure your DataMiner Agent:
 > [!IMPORTANT]
 > For security reasons, we strongly recommend creating a second user and disabling the built-in administrator account once the setup is complete. See [Managing users](xref:Managing_users).
 
+### WSL management
+
+If you chose for the Self-hosted - Local storage you are running a WSL container for your storage. This can be managed if you like.
```

Comment:
"This can be managed if you like" is rather vague. For which purpose(s) would a user do this?